test_setup.sh

```diff
@@ -8,5 +8,5 @@ do
   aws --no-sign-request s3 cp s3://brain-score-tests/tests/test_benchmarks/${f} ${SCRIPT_DIR}/tests/test_benchmarks/
 done
 
-pip install git+https://github.com/brain-score/model-tools --no-deps
+pip install git+https://github.com/brain-score/model-tools@core_refactor --no-deps
```

Comment:
once this qualification is removed, I think this looks good

Reply:
Of course, it’ll fail tests after that.

Reply:
so in what order do you want to merge? 1. brain-score, 2. model-tools?

Reply:
That works. Then I'll do a separate PR to remove the qualification.